src/lib/polkadotApi.ts

```diff
@@ -0,0 +1,114 @@
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { getBlockNumber } from "./connections";
+import { Keyring } from "@polkadot/api";
+import { GENESIS_HASHES } from "./connections";
+import {
+    storeBlockNumber,
+    storeValidAccounts,
+    storeConnected,
+    storeToken,
+    dotApi
+} from "./stores";
+import { options } from "@frequency-chain/api-augment";
+
+import type { DotApi } from "$lib/storeTypes";
+import type { KeyringPair } from "@polkadot/keyring/types";
+import type { web3Enable, web3Accounts } from "@polkadot/extension-dapp";
+import type { InjectedAccountWithMeta } from "@polkadot/extension-inject/types";
+import type { ChainProperties } from "@polkadot/types/interfaces";
+
+interface AccountMap {
```

Comment:
Unless you're going to implement functions on these two, or going to add properties to them, I think it's simpler to define type aliases instead, i.e. 
```typescript
type AccountMap = Record<string, KeyringPair>;
type MetaMap = Record<string,InjectedAccountWithMeta>
```
Unions will work with this too.
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#interfaces